tests/test_parametric_components/test_ToroidalFieldCoilTripleArc.py

```diff
@@ -32,8 +32,21 @@ def test_ToroidalFieldCoilTripleArc_rotation_angle(self):
             distance=50,
             number_of_coils=8,
         )
-        test_volume = test_shape.volume
 
+        test_shape.rotation_angle = 360
+        test_shape.workplane = "XZ"
+        test_volume = test_shape.volume
         test_shape.rotation_angle = 180
+        assert test_shape.volume == pytest.approx(test_volume * 0.5, rel=0.01)
 
+        test_shape.rotation_angle = 360
+        test_shape.workplane = "YZ"
+        test_volume = test_shape.volume
+        test_shape.rotation_angle = 180
         assert test_shape.volume == pytest.approx(test_volume * 0.5, rel=0.01)
+
+        # test_shape.rotation_angle = 360
```

Comment:
are these commented out lines needed

Reply:
These commented out tests are required, but will only pass when #308 is resolved